### DISCUSSION TITLE
Is it possible to change the default location a Daily note is created in?

### DISCUSSION THREAD

**Discussion by angelechoes** [702573N/Obsidian-Tasks-Calendar#48] [Q&A]
From calendar, when you click on a day for which you don't have a note it will create it under the "tasksCalendar" folder.  I would like it to go under the same location I have specified for other Daily Notes (using the core Daily Notes plugin)

**Comment by 702573N** [comment]
With the parameter `dailyNoteFolder: "My Daily Note Folder"` you can set a default location for your daily notes.

For example:

```dataviewjs
await dv.view("tasksCalendar", {pages: "", view: "month", firstDayOfWeek: "1", dailyNoteFolder: "Daily Notes", options: "style1 noLineClamp"})
```

**Comment by 702573N** [comment]
https://github.com/702573N/Obsidian-Tasks-Calendar#dailynotefolder
